src/snowflake/connector/pandas_tools.py

```diff
@@ -32,13 +33,13 @@
 logger = getLogger(__name__)
 
 
-def chunk_helper(lst: T, n: int) -> Iterator[tuple[int, T]]:
+def chunk_helper(lst: pd.DataFrame, n: int) -> Iterator[tuple[int, pd.DataFrame]]:
```

Comment:
We can't import and use pandas directly, because python connector doesn't install pandas by default. I guess you have to
```suggestion
def chunk_helper(lst: "pandas.DataFrame", n: int) -> Iterator[tuple[int, "pandas.DataFrame"]]:
```

Reply:
Good catch! I see, just saw in the same file `pandas.DataFrame` is used. Going to try that out.